aries-site/src/data/wcag/components.json

```diff
@@ -10,8 +10,25 @@
       { "rule": "2.4.4", "status": "conditional" },
       { "rule": "2.4.7", "status": "passed" },
       { "rule": "2.4.9", "status": "conditional" },
+      { "rule": "2.4.13", "status": "failed" },
       { "rule": "3.2.1", "status": "passed" },
       { "rule": "4.1.2", "status": "passed" }
+    ],
+    "menu": [
```

Comment:
Can we have the rules sorted in numerical order so that they display in the same order they would on the wcag site?

Reply:
done